packages/paste-core/components/slider/src/Slider.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import {Box, type BoxProps} from '@twilio-paste/box';
+import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+import {useSliderState, useSlider, useSliderThumb} from '@twilio-paste/react-spectrum-library';
+
+import {SliderThumb} from './SliderThumb';
+import {StyledTrack} from './StyledTrack';
+
+export interface SliderProps {
+  element?: BoxProps['element'];
+  id: string;
+  disabled?: boolean;
+  hasError?: boolean;
+  hideRangeLabels?: boolean;
+  minValue?: number;
+  maxValue?: number;
+  step?: number;
+  /**
+   * Used to adjust how the numbers are rendered and interpreted.
+   * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat \
+   */
+  numberFormatter: Intl.NumberFormat;
+  value?: number;
+  onChange?: (value: number) => void;
+  /** Fired when the slider stops moving, due to being let go. */
+  onChangeEnd?: (value: number) => void;
+}
+
+export const Slider: React.FC<SliderProps> = (props) => {
+  const inputRef = React.useRef(null);
+  const trackRef = React.useRef(null);
```

Comment:
question: shouldn't we be forwarding and merging at least one of these refs for a consumer?